app/views/layouts/application.html.erb

```diff
@@ -36,6 +36,7 @@
     )%>);
 
     new ELMO.Views.MissionChangeDropdown({el: '#change_mission select'});
+    new ELMO.Views.RegenerableFieldView();
```

Comment:
Ah, good point. I think it would work to put it in the method output as you suggest?